### PR TITLE
DDF-4801 Modify query advanced view to be more vertical

### PR DESCRIPTION
**Pull request by bellcc**
#### What does this PR do?
- Modify the query advanced view to appear more vertical which enables it to fit into smaller containers. 

#### Who is reviewing it? 

@cantstoptheunk 
@GabrielFabian 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@tbatie

#### How should this be tested?

- Verify the existing advanced search functionality. 

#### Any background context you want to provide?
- None

#### What are the relevant tickets?
For GH Issues:
Fixes: #4801 

#### Screenshots
Screen Shot 2019-05-07 at 8 32 38 AM

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js [hunk 1]
```diff
@@ -117,24 +117,26 @@ module.exports = Marionette.LayoutView.extend({
...
-          options={this.options}
-        />
+        <div className="filter-region global-bracket is-left">
+          <div
+            className="filter-attribute"
+            data-help="Property to compare against."
+          />
+          <div
+            className="filter-comparator"
+            data-help="How to compare the value for this property against
```
Comment (cantstoptheunk) [inline ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js]: ✏️ Could shorten by saying
`How to compare the property value against the provided value`

**Comment by bellcc** [comment]
build now

**Comment by GabrielFabian** [comment]
hero in progress

**Comment by GabrielFabian** [comment]
hero successful! LGTM

**Pull request closed without merging**